h/sentry/subscribers.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import unicode_literals
```

Comment:
Our linting still requires these on all files

Reply:
Note there's a [ticket](https://github.com/hypothesis/h/issues/5715) to remove these in this sprint-looks like this will prob be merged first but you never know.

Reply:
Removed these